src/executables/util_cs.c

```diff
@@ -3343,6 +3399,7 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
       if (error != NO_ERROR)
 	{
 	  PRINT_AND_LOG_ERR_MSG ("%s\n", db_error_string (3));
+	  goto print_applyinfo_usage;
 	}
       error = NO_ERROR;
```

Comment:
this one too. as you add `goto print_applyinfo_usage` above, this line is not required.

Reply:
I will modify the code same as like above comment 

[above comment](https://github.com/CUBRID/cubrid/pull/3374#discussion_r813747265)

Reply:
`goto print_applyinfo_usage` is deleted. So I clicked 'Resolve conversation'